tests/std/tests/P2445R1_forward_like/test.compile.pass.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <type_traits>
+#include <utility>
+
+using namespace std;
+
+struct T {}; // class type so const-qualification is not stripped from a prvalue
+using CT = const T;
+using U  = int;
+using CU = const U;
+
+T t{};
+const T& ct = t;
+
+static_assert(is_same_v<decltype(forward_like<U>(T{})), T&&>);

```

Comment:
It would be nice here to swap `U` and `T`, so that `U` corresponds to `_Uty` and `T` corresponds to `_Ty`.

Reply:
I missed renaming of  `t` and `ct`. Will do later.